code/modules/clothing/glasses/glasses.dm

```diff
@@ -395,6 +395,13 @@
 	tint = 3				//to make them blind
 	prescription_upgradable = 0
 
+/obj/item/clothing/glasses/blindfold_fake
+	name = "tattered blindfold"
+	desc = "A see-through blindfold perfect for cheating at games like pin the stunbaton on the clown."
+	icon_state = "blindfold"
+	item_state = "blindfold"
+	prescription_upgradable = 0
```

Comment:
```suggestion
	prescription_upgradable = FALSE
```

Reply:
Ah I made it =0 like normal blindfolds have because what was suggested before was a bit conflicting in the words. I'll do this though.